library/std/src/sys/unix/ext/process.rs

```diff
@@ -319,3 +320,70 @@ impl IntoRawFd for process::ChildStderr {
 pub fn parent_id() -> u32 {
     crate::sys::os::getppid()
 }
+
+/// Converts a signal number to a string representation suitable for a human reader.
+///
+/// This does not necessarily produce the same output as the C `strsignal` function.
+/// Currently there are two deviations from the return value of `strsignal`:
+///
+///  * `signal_describe` does not translate the string according to the locale; C `strsignal`
+///    usually does.  Unfortunately a translated version is not currently available.
```

Comment:
None of Rust's standard library functions currently include any kind of translation; this function is not unusual in that regard. (The first sentence, stating that it *isn't* translated, is helpful; the "Unfortunately" could be attached to many other functions, and is a known limitation of std.)